### PR TITLE
Fix(click): Clicking inside a block with hyperlink works on first attempt

### PR DESCRIPTION
**Pull request by sid597**
PR  for #1213

### CODE REVIEW

**Comment by sid597** [comment]
There is a bug in the current implementation due to which block-ref styles are not applied. This PR is more of a draft to show my current approach.

**Comment by sid597** [comment]
Fixed the issue by using pointer-events, tested on a few custom blocks which contain all type of elements.

File: test/athens/parser/compatibility_test.clj [hunk 1]
```diff
@@ -269,7 +269,7 @@
 
     [:block
      [:paragraph
-      [:span
+      [:testRun
```
Comment (neotyk) [inline test/athens/parser/compatibility_test.clj:272]: it should be `textRun`

**Review by neotyk — CHANGES_REQUESTED** [review]
Using CamelCase is not very idiomatic Clojure, please use kebab-case for var, fn, keyword names.
In this particular instance `:textRun` -> `:text-run`.

Apart from it LGTM, Thank you :-)

**Review by tangjeff0 — CHANGES_REQUESTED** [review]
see Alex's requests

File: test/athens/parser/compatibility_test.clj (hunk 1 above)
Comment (sid597) [reply test/athens/parser/compatibility_test.clj]: Aha!, Thank you, Alex.

**Comment by sid597** [comment]
Thank you Alex for reviewing this, more PR's coming your way from my side :)

**Review by tangjeff0 — APPROVED** [review]

**Pull request merged**